# Goal - Number of tilt cycles to perform
int32 tilt_cycles
---
# Result - Success result & final tilt position
bool success
float64 tilt_position
---
# Feedback - Number of tilt cycles performed & current tilt position
int32 progress
float64 tilt_position
